src/toolbox/utils/to-bool.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { toBool } from "./to-bool";
+import { expect } from "chai";
+import "mocha";
+
+const tests = [
+  [1, true],
+  [{}, true],
+  [[], true],
+  ["asdf", true],
+  [null, false],
+  [undefined, false],
+  [0, false]
```

Comment:
Let's add a couple tests for the booleans themselves, and make sure strings aren't returned as the boolean values they're written.
[true, true]
[false, false]
["true", true]
["false", false]